src/braket/circuits/gates.py

```diff
@@ -279,9 +278,8 @@ def gphase(
 
         Unitary matrix:
 
-            .. math:: \mathtt{gphase}(\gamma) = e^{i \gamma} I = \begin{bmatrix}
-                        e^{i \gamma} & 0 \\
-                        0 & e^{i \gamma} \end{bmatrix}.
```

Comment:
Some of the backslashes in this section are incorrectly getting formatted:
https://github.com/amazon-braket/amazon-braket-sdk-python/pull/946/files#diff-85b1f0c3121c7b89a6f7951f13b6ef494dca0216445ef035fe69ed30e054fdc2L293

Reply:
This may be due to the comment block being `r"""`. In that case, I think removing the `\\` may work for the [doc](https://amazon-braket-sdk-python--946.org.readthedocs.build/en/946/_apidoc/braket.circuits.gates.html#braket.circuits.gates.GPhase.gphase)

Reply:
I agree, although this appears to be the case for every single gate in the module, not just this one.